### ISSUE TITLE
[Bug]: counter cannot decrease in value

### ISSUE THREAD

**Issue by liang09255** [milvus-io/milvus#35767]
### Is there an existing issue for this?

- [x] I have searched the existing issues

### Environment

```markdown
- Milvus version: 2.4.7
- Deployment mode(standalone or cluster): standalone
- MQ type(rocksmq, pulsar or kafka):    
- SDK version(e.g. pymilvus v2.0.0rc2): attu2.4.9
- OS(Ubuntu or CentOS): openeuler
- CPU/Memory: 8h 16g
- GPU: 
- Others:
```

### Current Behavior

1. docker-compose deploy milvus on openeuler20.03
2. local attu connected to remote milvus
3. build collection and insert entity success
4. search falied and milvus panic

### Milvus Log

2024/08/28 07:54:27 http: panic serving 10.131.114.69:64201: counter cannot decrease in value
goroutine 5392 [running]:
net/http.(*conn).serve.func1()
        /usr/local/go/src/net/http/server.go:1868 +0xb9
panic({0x52fa4e0?, 0xc003576ab0?})
        /usr/local/go/src/runtime/panic.go:920 +0x270
github.com/prometheus/client_golang/prometheus.(*counter).Add(0xc003602000?, 0xc0035389e0?)
        /go/pkg/mod/github.com/prometheus/client_golang@v1.14.0/prometheus/counter.go:109 +0xe9
github.com/milvus-io/milvus/internal/distributed/proxy.(*Server).startHTTPServer.func1(0xc003602000)
        /workspace/source/internal/distributed/proxy/service.go:197 +0x551
github.com/gin-gonic/gin.(*Context).Next(...)
        /go/pkg/mod/github.com/gin-gonic/gin@v1.9.1/context.go:174
github.com/gin-gonic/gin.serveError(0xc003602000, 0x194, {0x869d800, 0x12, 0x12})
        /go/pkg/mod/github.com/gin-gonic/gin@v1.9.1/gin.go:656 +0x4b
github.com/gin-gonic/gin.(*Engine).handleHTTPRequest(0xc00256e4e0, 0xc003602000)
        /go/pkg/mod/github.com/gin-gonic/gin@v1.9.1/gin.go:649 +0x485
github.com/gin-gonic/gin.(*Engine).ServeHTTP(0xc00256e4e0, {0x6056290?, 0xc0029b8380}, 0xc002205f00)
        /go/pkg/mod/github.com/gin-gonic/gin@v1.9.1/gin.go:576 +0x1dd
net/http.serverHandler.ServeHTTP({0xc003daf1a0?}, {0x6056290?, 0xc0029b8380?}, 0x6?)
        /usr/local/go/src/net/http/server.go:2938 +0x8e
net/http.(*conn).serve(0xc002bbd050, {0x606d000, 0xc0015ecb70})
        /usr/local/go/src/net/http/server.go:2009 +0x5f4
created by net/http.(*Server).Serve in goroutine 1630
        /usr/local/go/src/net/http/server.go:3086 +0x5cb

**Comment by liang09255** [comment]
downgraded version to 2.3.21 ok

**Comment by yanliang567** [comment]
@liang09255 what is 10.131.114.69:64201 for?  I don't think milvus has this port by default.  
@tedxu did you changed any dependencies for 2.4.x ?

/assign @liang09255 
/unassign

**Comment by congqixia** [comment]
This is a caused by `Adding` negative written size in restful API. I shall file a fix this ASAP

**Comment by shivabohemian** [comment]
The same problem in 2.4.9.

**Comment by congqixia** [comment]
This fix is merged and release with v2.4.10. Could you please check whether v2.4.10 solves your problems. thanks in advance

**Comment by liang09255** [comment]
> This fix is merged and release with v2.4.10. Could you please check whether v2.4.10 solves your problems. thanks in advance

ok

/unassign
